pkg/providers/hashicorp.go

```diff
@@ -90,7 +80,8 @@ func (g *hashiCorp) Fetch() (*File, error) {
 		log.Infof("Getting %s release for %s", g.tag, g.repo)
 		release, err = g.getRelease(g.repo, g.tag)
 	} else {
-		version, _, err := g.GetLatestVersion()
+		var version string
+		version, _, err = g.GetLatestVersion()
```

Comment:
I don't understand the reasoning behind this change. What was the linting error that would recommend this?

Reply:
Same problem as in gitlab provider explained above.